tests/uefistored/test_auth_var.py

```diff
@@ -62,3 +65,53 @@ def test_auth_variable(imported_vm):
 
     # Set the variable with new data, signed by a different cert
     set_and_assert_var(vm, Certificate(), b'this should fail', should_pass=False)
+
+    vm.shutdown(verify=True)
```

Comment:
The test this snippet belongs to (github won't let me comment the lines higher) should probably depend on the linux_uefi_vm fixture too, and the `if vm.is_windows` check be removed from it.